alderaan/property_files/prometheus.properties

```diff
@@ -0,0 +1,20 @@
+JUMP_HOST=
+USER=root
+SETUP_PBENCH=false
```

Comment:
SETUP_PBENCH, ENVIRONMENT, TOOLING_INVENTORY are not needed. We will handle the pbench setup using a separate job ( SETUP-TOOLING ) which is going to run just once after openshift install and we will use the same tools for all the tests.